test/everypolitician/popolo/event_test.rb

```diff
@@ -27,4 +31,24 @@ def test_accessing_event_properties
     assert_equal '2014-11-27', event.start_date
     assert_nil event.end_date
   end
+
+  def test_event_people_returns_array_of_person_objects
+    event = popolo_wales.events.map { |e| e }[4]
```

Comment:
Ah, from further reading, I'm guessing you can't take `events.first` because you want a Term, and that's an Election. Both the test name and the set-up of event should make that clear (e.g. with `popolo_wales.events.where(classification: 'legislative period').first`)